### ISSUE TITLE
Section on deailing with extracts and data products

### ISSUE THREAD

**Issue by maning** [hotosm/learnosm#111]
From Mikel:

I'd also add a section to dealing with extracts and data products ... ie finding stuff on GeoFabrik, using Overpass Turbo.

Reference: http://lists.openstreetmap.org/pipermail/hot/2013-July/003370.html

**Comment by mikelmaron** [comment]
Oldie but goodie!

Good stuff @maning. This could use a refresh, now that there's a HOT export tool relaunch. And Overpass Turbo is a key tool to include too.

What do you think @russdeffner? Someone on the Training Working Group willing to help with this?

**Comment by mikelmaron** [comment]
There's actually already great stuff here to point to for Overpass

https://www.mapbox.com/guides/overpass-turbo/

**Comment by russdeffner** [comment]
Yes, I think we should look at expanding and/or combining http://learnosm.org/en/osm-data/getting-data/ and http://learnosm.org/en/osm-data/geofabrik-and-hot-export and include information on Overpass and maybe other export/extracts like Garmin, etc.
